Automattic/jetpack
# Publicize: Initialize the Publicize classes based on the Publicize Module active status.

#### Changes proposed in this Pull Request:

* Load Publicize classes only if the Publicize Module is active
* Changed the setup code of Publicize package to initialize the Publicize classes based on the `jetpack_feature_publicize_enabled` hook. 

#### Jetpack product discussion


p1653045349526599/1653019151.332719-slack-CBG1CP4EN
#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Install Jetpack and make sure Publicizing works correctly
* Deactivate the Publicize module and make sure all the Publicize elements get hidden on the block editor. 
* Activate the Publicize module again and make sure you're able to Publicize correctly. 
* Deactivate the Publicize module and deactivate Jetpack, activate the Jetpack Social Plugin and make sure you're able to  Publicize via the Classic Editor.
